src/components/TimeZoneSelector.tsx

```diff
@@ -40,7 +40,16 @@ export default function TimeZoneSelector(props: TimeZoneSelectorProps): JSX.Elem
     return optionA?.value === optionB?.value;
   };
 
-  return (
+  return disabled ? (
+    <Textfield
+      id='timezones2'
```

Comment:
missed this earlier.. can we call it 'time-zone-selector-disabled' and below 'time-zone-selector'? or anything else that is more clarifying?

Reply:
Yes. Not sure why this was the id before